src/parsers/directive.parser.ts

```diff
@@ -18,7 +18,10 @@ export class DirectiveParser extends AbstractTemplateParser implements ParserInt
 		template = this._normalizeTemplateAttributes(template);
 		$(template)
 			.find('[translate]')
-			.each((i: number, element: CheerioElement) => {
+            .contents() // get child nodes
+            .filter(function() { // only keep text nodes
+				return this.nodeType === 3; // node type 3 = text node
```

Comment:
You might have a point there. I've pushed some code to extract direct text nodes only for now. Maybe we can come up with something in the future to allow it.